receipts/handler/handler.go

```diff
@@ -95,15 +93,12 @@ func (r *ReceiptHandler) GetPendingTxHashList() [][]byte {
 	return hashListCopy
 }
 
-func (r *ReceiptHandler) Close() error {
-	if err := r.leveldbReceipts.Close(); err != nil {
-		return errors.Wrap(err, "closing receipt leveldb")
-	}
-	return nil
+func (r *ReceiptHandler) Close() {
+	r.evmAuxStore.Close()
 }
 
 func (r *ReceiptHandler) ClearData() error {
-	r.leveldbReceipts.ClearData()
+	r.evmAuxStore.ClearData()
```

Comment:
Lets get rid of this `ReceiptHandler.ClearData()` completely, can just call `os.RemoveAll` directly where needed.

Reply:
Since we use TM golevel db, we can easily use dbm.MemDB so most of the tests don't need to clear data anymore.